app/views/runs/_quick_stats.slim

```diff
@@ -0,0 +1,53 @@
+- previous_upload = @run.user.try(:previous_upload_for, @run.category)
```

Comment:
Looks like the delta arrows appear when the run you're looking at is the second best you have on record.

![chrome_2018-06-06_19-26-09](https://user-images.githubusercontent.com/7030913/41075012-930719ca-69bf-11e8-8182-ccfc97ae6ff6.png)